File: Practice/08/C++/example8/example8/example8.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
    int stop = 1;
    double x, y, result;
    char op;
    setlocale(LC_ALL, "Russian");

    for (;stop == 1;) {
        stop = 0;

        cin >> x >> op >> y;
        if (op == '+') {
            result = x + y;
            cout << result;
        }
        else if (op == '-') {
            result = x - y;
            cout << result;
        }
        else if (op == '*') {
            result = x * y;
            cout << result;
        }
        else if (op == '/') {
            if (y == 0) {
                cout << "Знаменатель не может быть равен 0" << endl;
                stop = 1;
            }
            else {
                result = x / y;
                cout << result;
            }
            }
        else {
            cout << "В неверном формате введены данные." << endl;
        }


    }
}

File: Practice/08/Python/example8.py

stop = 1

while stop == 1:
	stop = 0
	x, op, y = map(str, input().split())
	if (op == '+'):
	    result = float(x) + float(y)
	    print(result)
	elif (op == '-'):
	    result = float(x) - float(y)
	    print(result)
	elif (op == '*'):
	    result = float(x) * float(y)
	    print(result)
	elif (op == '/'):
		if int(y) == 0:
			print("Знаменатель не может быть равен 0")
			stop = 1
		else:
		    result = float(x) / float(y)
		    print(result)
	else:
	    print("В неверном формате введены данные.")
